solana-programs/anchor/audius-data/lib/utils.ts

```diff
@@ -168,3 +172,64 @@ export const getContentNode = async (
     derivedAddress,
   };
 };
+
+/**
+ * Returns property value from CN shellEnv#.sh from local service-commands provisioning.
+ * 
+ */
+export const getContentNodeConfigValue = async ({ spId, key }: { spId: number, key: string }): Promise<string> => {
+  const spConfigFilePath = `creator-node/compose/env/tmp/shellEnv${spId}.sh`
+  let value;
+  try {
+    const spConfig = await fs.readFile(path.join(process.env.PROTOCOL_DIR, spConfigFilePath), "utf-8");
+    value = spConfig.split(`${key}=`)[1].split("\nexport")[0];
+  } catch (error) {
+    throw new Error(`Error getting private key from sp config file ${spConfigFilePath}: ${error}`);
+  }
+  return value;
+}
+
+/**
+ * converts hex eth pk value (eg 358edb5f358b697c32d3dd3c0107da568635334078a21bb755aaa41f084681ff) 
```

Comment:
mind shortening this so it doesn't map to a real public key? we try to avoid committing these if possible even if they are totally safe